src/routes/api/TBAEventKeys/+server.ts

```diff
@@ -3,7 +3,7 @@ import { error } from "@/consoleUtils"
 import { json, type RequestHandler } from "@sveltejs/kit"
 
 export const GET: RequestHandler = async ({ request: _ }: any) => {
-    const year = "2025"
+    const year = "2022"
```

Comment:
Change to 2025 before merging.
```suggestion
    const year = 2025
```